app/src/models/github-repository.ts

```diff
@@ -52,6 +52,10 @@ export class GitHubRepository {
   }
 }
 
+export type ForkedGitHubRepository = GitHubRepository & {
+  parent: GitHubRepository
+}
```

Comment:
i think `parent` should be readonly here too.

we could also explicitly say that `fork` will return `true` in the typing here. i don't think typescript will figure that out on its own

```suggestion
export type ForkedGitHubRepository = GitHubRepository & {
  readonly parent: GitHubRepository
  readonly fork: true
}
```

Reply:
Good catch! I was thinking about whether we can remove the `fork` property now and use everywhere the new `isRepositoryWithForkedGitHubRepository` method (which is preferred since it will narrow down the type), but in the meantime it's good to have `fork: true` in the type!